dlhub_sdk/client.py

```diff
@@ -336,6 +347,63 @@ def get_result(self, task_id, verbose=False):
             result = result[0]
         return result
 
+    def easy_publish(self, title: str, creators: Union[str, list[str]], short_name: str, servable_type: str, serv_options: dict[str, Any]):
+        """Simplified publishing method for servables
+
+        Args:
+            title (string): title for the servable
+            creators (string | list): either the creator's name (FamilyName, GivenName) or a list of the creators' names
+            short_name (string): shorthand name for the servable
+            servable_type (string): the type of the servable, must be a member of ("static_method",
+                                                                                   "class_method",
+                                                                                   "keras",
+                                                                                   "pytorch",
+                                                                                   "tensorflow",
+                                                                                   "sklearn")
+            serv_options (dict): the servable_type specific arguments that are necessary for publishing
+        Returns:
+            (string): task id of this submission, can be used to check for success
+        Raises:
+            ValueError: If the given servable_type is not in the list of acceptable types
+            Exception: If the serv_options are incomplete or the request to publish results in an error
+        """
+        # conversion table for model string names to classes
+        models = {"static_method": PythonStaticMethodModel,
+                  "class_method": PythonClassMethodModel,
+                  "keras": KerasModel,
+                  "pytorch": TorchModel,
+                  "tensorflow": TensorFlowModel,
+                  "sklearn": ScikitLearnModel}
+
+        # raise an error if the provided servable_type is invalid
+        model = models.get(servable_type)
+        if model is None:
+            raise ValueError(f"dl.easy_publish given invalid servable type: {servable_type}, please refer to the docstring")
+
+        # attempt to construct the model and raise a helpful error if needed
+        try:
+            # if the servable is a python function, set the parameter to attempt to auto-generate the inputs/outputs
+            if servable_type in {"static_method", "class_method"}:
+                serv_options["auto_inspect"] = True
+
+            model_info = model.create_model(**serv_options)
+        except Exception as e:
+            help_err = HelpMessage(f"Help can be found here:\nhttps://dlhub-sdk.readthedocs.io/en/latest/source/dlhub_sdk.models.servables.html#"
+                                   f"{model.__module__}.{model.__name__}.create_model")
+            raise help_err from e
+
+        # set the required datacite fields
```

Comment:
what if the user wants to set other datacite fields? it appears that there isn't currently functionality for that in this method, is that correct?

Reply:
that is correct. the plan has been to add an additional dict parameter to pass those in, but it has been put off for the time being.

Reply:
great, thanks for clarifying. @isaac-darling could you check if an issue has been made for that, and if not, could you create it and link it in a comment on this PR?